src/chrome/content/rules/3DCenter.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="3DCenter.org (cacert)" platform="cacert">
+	<target host="www.3dcenter.org" />
+	<target host="m.3dcenter.org" />
+	<rule from="^http://www\.3dcenter\.org/" to="https://www.3dcenter.org/"/>
+	<rule from="^http://m\.3dcenter\.org/" to="https://m.3dcenter.org/"/>
```

Comment:
It is more easy to understand if you replace the two rules for www.3dcenter.org and m.3dcenter.org by `<rule from="^http:" to="https:" />`.  There is a new [Ruleset Style Guide](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md).